### ISSUE TITLE
[Feature] In Particular hierarchy of tree only one node should be opened.When one more node is opened,the already opened node is closed.

### ISSUE THREAD

**Issue by soundhar07** [apache/echarts#18564]
### What problem does this feature solve?

In Particular hierarchy of tree only one node should be opened.When one more node is opened,the already opened node is closed.In this case, the tree will be looking neat and clean.

![image](https://user-images.githubusercontent.com/38309163/234206209-042659f3-98a4-4c3c-8846-00116b31240d.png)

### What does the proposed API look like?

In Particular hierarchy of tree only one node should be opened.When one more node is opened,the already opened node is closed.
![image](https://user-images.githubusercontent.com/38309163/234206474-abb5271d-9e0e-4ef1-8522-560ef9511f1c.png)

**Comment by helgasoft** [comment]
yes, interesting feature. Currently there is no dispatch [action](https://echarts.apache.org/en/api.html#action) to open/close tree nodes.

**Comment by tetratrionofficial** [comment]
is there any update regarding this ?
